src/domain/books/types.ts

```diff
@@ -1,13 +0,0 @@
-import { DefaultLoadingState } from "../../types";
-
-export type Book = {
-  id: string;
-  name: string;
-  author: string;
-  description: string;
-  price: number;
-}
-
-export type BooksState = DefaultLoadingState & {
-  books: Book[];
-}
```

Comment:
All of the types related to books belong here.

Reply:
Sure Sir I can make this change